tools/scripts/server/startServer.bat

```diff
@@ -1,30 +1,37 @@
 :: This script starts the local arma dev-server
-:: All meta-information are set via the setServerData.bat script that
+:: All meta-information are set via the setMetaData.bat script that
 :: is being called from this one
 @echo off
 
 :: set meta infos
-call %~dp0.\setMetaData.bat
+call "%~dp0.\setMetaData.bat"
+
+:: convert relative to absolute path of clib
+:: so armaserver can read it properly
+pushd "%~dp0..\..\..\PBOs\dev\@CLib"

```

Comment:
Before this line there should be a check whether the directory actually exists. If not, exit with a useful error message (like "Can't find mod-Dir - did you forget to build the mods before?")